sdk/iOS/src/MSPullSettings.h

```diff
@@ -0,0 +1,29 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import <Foundation/Foundation.h>
+
+/// Settings that control the pull behavior
+@interface MSPullSettings : NSObject
+
+#pragma mark * Properties
+
+///@name Properties that control pull behavior
+///@{
+
+/// Size of pages requested from the server while performing a pull
+@property (nonatomic) NSInteger pageSize;
+
+///@}
+
+#pragma mark * Class methods
+
+///@name Class factory methods
```

Comment:
nit: this is not in the style of apple documentation, should be more action-y, like Creating a MSPullSetting.